src/components/Button/LinkButton.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { ComponentRegister } from "../../ComponentRegister";
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link LinkButton}
+ */
+export interface LinkButtonProps extends BaseButtonProps {
+  href: string;
+  children: React.ReactNode;
+}
+
+/**
+ * The link button, styled as a button, that can be used for routing
+ * within the host application.
+ */
+export const LinkButton: React.FunctionComponent<LinkButtonProps> = (
+  props: LinkButtonProps
+): JSX.Element => {
+  const { id, className, isStartButton, href, children } = props;
+  return (
+    <ComponentRegister.Link
+      id={id}
+      className={classNames("govuk-button", className, {
+        "govuk-button--start": isStartButton
+      })}
```

Comment:
Shouldn't these also have `lbh-button` classes? I think we should probably be adding them whether or not the css actually references them.

This also brings up my feeling that it's a bit odd to have to have both `govuk-*` and `lbh-*`. Can't the `lbh-*` stylesheets alias the `govuk-*` ones? Probably a question for @LBHELewis.

Reply:
@erbridge I think for now we need the `govuk-*` and `lbh-*` classes, sorry. We don't have complete control over the pattern library markup and are heavily dependent on govuk templates and styling. It's also helpful for us to be able to see at a glance which code is "ours" and which is coming from govuk.

Reply:
Makes sense.